docs/api.md

```diff
@@ -123,7 +123,17 @@ the host.
 *hardwareProfile* -- The name of the hardware profile that matches the
 hardware discovered on the host. Details about the hardware are saved
 to the *hardware* section of the status. If the hardware does not
-match a known profile, the value "unknown" is used.
+match a known profile, the value "unknown" is used.  In practice, so far
+this affects which device the deployed OS image is written to.  The
+following `hardwareProfile` settings and the corresponding root device
+are listed here, and are definitely subject to change.
+
+| **hardwareProfile** | **Root Device** |
+|---------------------|-----------------|
+| `default`           | /dev/sda        |
```

Comment:
Should this say `unknown` per line 126?

Reply:
It’s mapped to default in the installer.  It seemed odd to me to expose “unknown” to the user.

Reply:
Should we change line 126 then? It's very confusing to say that the default is some value not in the table and then give a table that includes a `default` entry.

Reply:
I think your original comment is right, for BMO docs, `unknown` should be the value on line 133. Perhaps with an asterisk or something that indicates it's the default value

Reply:
good catch, - fixed